test/unit/modules/helpers/get-dom-node-profile-spec.js

```diff
@@ -1,60 +1,140 @@
 import { getDomNodeProfile } from '../../../../lib/helpers/getDomNodeProfile';
 
-describe('Keen.helpers.getDomNodeProfile', () => {
-  const element = {
-    action: '/submit',
+describe('ProdPerfectRecorder.helpers.getDomNodeProfile', () => {
+  const elP = document.createElement('p');
+  elP.setAttribute('aria-foo', 'yes')
+  elP.setAttribute('class', 'styled');
+  elP.setAttribute('for', 'ProdPerfectRecorder');
+  elP.setAttribute('id', 'myID');
+  elP.setAttribute('ng-click', 'ngClickAttr');
+  elP.setAttribute('ng-model', 'ngModelAttr');
+  elP.setAttribute('title', 'myTitle');
+  elP.name = 'testEl';
+  elP.text ='myText';
+  elP.textContent ='myTextContent';
+  elP.style.cursor = 'pointer';
+
+  const expectedP = {
+    action: undefined,
+    'aria-foo': 'yes',
     className: 'styled',
-    clientTop: 2,
-    href: '#foo',
+    clientTop: null, // requires hack that I couldn't get working to work with JSDom
+    cursor: 'pointer',
+    href: null,
+    for: 'ProdPerfectRecorder',
     id: 'myID',
-    method: 'post',
+    method: undefined,
     name: 'testEl',
     'ng-click': 'ngClickAttr',
     'ng-model': 'ngModelAttr',
-    nodeName: 'myNode',
-    offsetLeft: 1,
+    nodeName: 'P',
+    offsetLeft: null, // requires hack that I couldn't get working to work with JSDom
     parentNode: null,
-    tagName: 'form',
+    tagName: 'P',
     text: 'myText',
     textContent: 'myTextContent',
     title: 'myTitle',
-    type: 'myType',
-    getAttribute: function(attr) {
-      return this[attr];
-    }
+    type: undefined
   }
 
-  test('should return relevant properties of the passed element, without text_content when not requested', () => {
-    expect(getDomNodeProfile(element)).toEqual({
-      action: element.action,
-      class: element.className,
-      href: element.href,
-      id: element.id,
-      method: element.method,
-      name: element.name,
-      ng_click: element['ng-click'],
-      ng_model: element['ng-model'],
-      node_name: element.nodeName,
+  const elA = document.createElement('a');
+  elA.setAttribute('href', '#foo');
+  elA.setAttribute('title', 'myATitle');
+  elA.setAttribute('type', 'submit');
+  elA.setAttribute('value', 'myAVal');
+  elA.method = 'post';
+  elA.action = '/my_action';
+  elA.name = 'testA';
+  elA.textContent ='myATextContent';
+
+  const expectedA = {
+    action: '/my_action',
+    className: '',
+    clientTop: null, // requires hack that I couldn't get working to work with JSDom
+    href: 'http://localhost:8080/#foo',
+    href_short: '#foo',
+    id: '',
+    method: 'post',
+    name: 'testA',
+    'ng-click': null,
+    'ng-model': null,
+    nodeName: 'A',
+    offsetLeft: null, // requires hack that I couldn't get working to work with JSDom
+    parentNode: null,
+    tagName: 'A',
+    text: 'myATextContent',
+    textContent: 'myATextContent',
+    title: 'myATitle',
+    type: 'submit',
+    valueAttr: 'myAVal'
```

Comment:
Ah, nice -- so we wont get any `value`-like attributes and dodge any potentially damning PII data.